magda-content-api/src/createApiRouter.ts

```diff
@@ -26,7 +36,7 @@ export default function createApiRouter(options: ApiRouterOptions) {
 
     /**
      * @apiGroup Content
-     * @api {post} /v0/content/:contentId Get All
+     * @api {post} /v0/content/all Get All
      * @apiDescription Get a list of content items and their type.
      * You must be an admin for this.
      *
```

Comment:
Can we document the filtering/inlining and how it's intended to be used here? Looking at the way the frontend uses it looks pretty powerful but I'm not exactly sure how it works api-wise.